<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>541</width>
      <height>501</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string notr="true">1</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="insertButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>20</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>insert</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>20</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>1</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="insertBox">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>20</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>2147483647</number>
    </property>
   </widget>
   <widget class="QPushButton" name="atButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>60</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>at</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="atBox">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>60</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>2147483647</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="atLine">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>60</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="removeButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>100</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>remove</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="removeBox">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>100</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>2147483647</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
